examples/ecscluster/index.ts

```diff
@@ -0,0 +1,34 @@
+import * as ecs from 'aws-cdk-lib/aws-ecs';
+import * as pulumi from '@pulumi/pulumi';
+import * as pulumicdk from '@pulumi/cdk';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as pulumiaws from "@pulumi/aws-native";
+
+class ECSClusterStack extends pulumicdk.Stack {
+    clusterArn: pulumi.Output<string>;
+
+    constructor(id: string, options?: pulumicdk.StackOptions) {
+        super(id, options);
+
+        const vpc = ec2.Vpc.fromLookup(this, 'MyVpc', {
+            isDefault: true,
+        })
+        const cluster = new ecs.Cluster(this, 'fargate-service-autoscaling', { vpc });
+
+        this.clusterArn = this.asOutput(cluster.clusterArn);
+
+        this.synth();
+    }
+}
+
+export const clusterArn = pulumiaws.getAccountId().then(account => {
+    const stack = new ECSClusterStack('teststack', {
+        props: {
+            env: {
+                region: pulumiaws.config.region,
+                account: account.accountId,
```

Comment:
It is more than a little unfortunate that we can't get the account id synchronously, but this was the cleanest way I could come up with doing this.

Reply:
I'm a bit late to this (coming from a closed ticket), but would an alternative be:

```typescript
export default async main() {
	const account = await pulumiaws.getAccountId();
	const { clusterArn }= new ECSClusterStack('teststack', {
		props: {
			env: {
				region: pulumiaws.config.region,
				account: account.accountId,
			},
		},
	});
	return {
		clusterArn
	};
}
```

Reply:
Indeed!  

I also tried to have this test use ESM and top-level await - using https://github.com/pulumi/pulumi/issues/5161#issuecomment-1076883623.  But that interacts poorly with `yarn link` as needed to run in tests (I think there must be some Node or Yarn bug here - but couldn't narrow it down).  It did work for me locally though - so is an option for users of this package - and allows this to be just:

```ts
const account = await pulumiaws.getAccountId();
const stack = new ECSClusterStack('teststack', {
    props: {
        env: {
            region: pulumiaws.config.region,
            account: account.accountId,
        }
    }
});
    
export const clusterArn = stack.clusterArn;
```